lib/src/client/call.dart

```diff
@@ -100,7 +101,13 @@ class ClientCall<Q, R> implements Response {
   bool isCancelled = false;
   Timer _timeoutTimer;
 
-  ClientCall(this._method, this._requests, this.options) {
+  final TimelineTask _timeline;
+
+  ClientCall(this._method, this._requests, this.options, this._timeline) {
+    _timeline?.start('rpc call created', arguments: {
+      'method': _method.path,
+      'timeout': options?.timeout?.toString(),
```

Comment:
should we be including any additional data here? are there other properties of _method we are interested in? what about _requests or _options?

Reply:
`_method` only contains three properties. `path` which basically is the name of rpc. `serializer` and `deserializer` functions are used to encode and decode data.

For `options`, three properties are there. `timeout` which we store here. `metadata` will be processed later, so it is recorded when we send request. `metadataProvider` is just a function which will modifies metadata. Is it needed?

`_requests` is a stream where users send data. It will be recorded in the _sendRequestt.

Reply:
got it. thanks for clarifying.